### PR TITLE
Updating build for 1es migration

### PR DESCRIPTION
**Pull request by soninaren**

### CODE REVIEW

File: eng/official-build.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      image: 1es-windows-2022
+      os: windows
+
+    stages:  
+    - stage: Build
+      dependsOn: []
+      jobs:
+      - template: eng/ci/templates/jobs/build.yml@self
+        parameters:
+          official: true
```
Comment (jviau) [inline eng/official-build.yml]: there is no `official` parameter declared by `build.yml`

File: eng/public-build.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  template: v1/1ES.Unofficial.PipelineTemplate.yml@1es
+  parameters:
+    pool:
+      name: 1es-pool-azfunc-public
+      image: 1es-windows-2022
+      os: windows
+
+    stages:
+    - stage: Build
+      dependsOn: []
```
Comment (jviau) [inline eng/public-build.yml:30]: This can be removed

File: eng/official-build.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1es
+  parameters:
+    pool:
+      name: 1es-pool-azfunc
+      image: 1es-windows-2022
+      os: windows
+
+    stages:  
+    - stage: Build
```
Comment (jviau) [inline eng/official-build.yml:44]: are you missing the call to `localization.yml` template?

File: eng/official-build.yml (hunk 1 above)
Comment (soninaren) [reply eng/official-build.yml]: Going to delete this parameter for now. Will use it to disable localization from public builds as discussed.

File: eng/official-build.yml (hunk 3 above)
Comment (soninaren) [reply eng/official-build.yml:44]: Have skipped it on purpose as it is broken. Will be adding it back in another PR once it is fixed

**Review by jviau — APPROVED** [review]

**Pull request merged**